lib/main_common.pm

```diff
@@ -1666,6 +1665,10 @@ sub load_extra_tests_phub {
     loadtest 'sysauth/sssd';
 }
 
+sub load_extra_tests_sdk {
+    loadtest 'console/gdb';
+}
+
```

Comment:
I guess that this would include the tests that are for specific packages that come from the sdk? Removing it from the console/extratests basically means that we will have that coverage lost, for other distros.

Reply:
I have taken care of extra_tests_in_textmode for Tumbleweed and Leap.
In the description are the verification runs and what needs to be done, adding sdk to EXTRATEST setting.

Reply:
The gdb module was only scheduled in load_extra_tests_console, so only on tests that BOOT_HDD_IMAGE and EXTRATEST=.*console.*

We can find those test suites and add EXTRATEST=.*console.*,sdk.*.